cf-execd/cf-execd.c

```diff
@@ -725,7 +736,7 @@ static void CFExecdMainLoop(EvalContext *ctx, Policy **policy, GenericAgentConfi
             }
         }
         /* 1 Minute resolution is enough */
-        MaybeSleepLog(LOG_LEVEL_VERBOSE, "Sleeping for pulse time %u seconds...", CFPULSETIME);
+        MaybeSleepLog(LOG_LEVEL_VERBOSE, "Sleeping for pulse time %u seconds...", GetPulseTime());
```

Comment:
The function should only be called once and the result should be used both above and here in the log message. Otherwise the log message can be misleading mentioning  a different value than the code actually does.

Reply:
It is only called once per version of the main loop: windows or !windows so seems OK to me.

Reply:
What Craig said. This is not just a log message, this is the windows version of the execd loop. 

+1 point to @craigcomstock for getting it right. -2 points to @vpodzime for not understanding his own code from 3 years ago, and -1 points to @larsewi for a wrong thumbsup :)

I've added some extra comments ;)

Reply:
Who cares about Windows?